source/xlsxreader.d

```diff
@@ -239,19 +222,19 @@ struct RowUntyped {
 		this.cur = this.startColumn;
 	}
 
-	@property bool empty() const pure nothrow @nogc @safe {
+	@property bool empty() const pure nothrow @nogc @safe scope {
 		return this.cur >= this.endColumn;
 	}
 
-	void popFront() pure nothrow @nogc @safe {
+	void popFront() pure nothrow @nogc @safe scope {
 		++this.cur;
 	}
 
-	inout(typeof(this)) save() inout pure nothrow @nogc @safe {
+	inout(typeof(this)) save() inout pure nothrow @nogc @safe return scope {
```

Comment:
Not a comment on the diff, but on D as a language.

"This line is just ugly."

Reply:
Yeah I fully agree.